### DISCUSSION TITLE
Consider adding gitter template for scala cli in Metals

### DISCUSSION THREAD

**Discussion by kpodsiad** [VirtusLab/scala-cli#1428] [General]
When I need to create a toy file I use scala cli.

When I need to have more than one file, I'm still using sbt because I can scaffold it in few second using `Metals new scala project` and giter8.

<img width="636" alt="Screenshot 2022-10-07 at 07 15 18" src="https://user-images.githubusercontent.com/37124721/194472892-6255392b-fe2c-46de-999e-f5570a4313e4.png">

It might be good to add a template which allows one to quickly create single module project with scala cli, just like https://github.com/VirtusLab/iskra for instance.

**Comment by Gedochao** [comment]
We actually have a ticket for this already: #19

**Reply by kpodsiad** [reply]
Thanks, I haven't noticed it earlier ;)
